frontend/src/pages/NewRunV2.tsx

```diff
@@ -72,11 +80,42 @@ function NewRunV2(props: NewRunV2Props) {
   const [isStartingNewRun, setIsStartingNewRun] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
   const [isParameterValid, setIsParameterValid] = useState(false);
+  const [clonedRuntimeConfig, setClonedRuntimeConfig] = useState<PipelineSpecRuntimeConfig>({});
+  const [usePipelineFromRunLabel, setUsePipelineFromRunLabel] = useState(
+    'Using pipeline from cloned run',
```

Comment:
I am thinking if this sentence correctly describes the use case. Do you mean `Using pipeline from existing run`?

Reply:
I think you are right. This is what I get from v1 codebase.